### ISSUE TITLE
Please add more info to the README about how to get image.php to work with zp_picturefill

### ISSUE THREAD

**Issue by JatB** [acrylian/zp_picturefill#10]
TL;DR
It would be great if the README had that for image.php, 

printDefaultSizedImage(getImageTitle())
needs to be changed to
printHDDefaultSizedImage(null, true)

to work and work without errors.

The long version:
When updating our template (a slightly modified Basic theme) with zp_picturefill, to get the thumbnails to work, all we really had to do was add “HD” to the functions you had listed. So, 

printAlbumThumbImage(getAnnotatedAlbumTitle())
in album.php became:
printHDAlbumThumbImage(getAnnotatedAlbumTitle())

All the thumbnails worked on all the pages and that was great, but when it came time to update image.php, that was when we ran into problems.

Changing
printDefaultSizedImage(getImageTitle());
to
printHDDefaultSizedImage(getImageTitle());

got us a page with no image. No rest of the page either, it just dropped off after where the image should be (and nothing was in the debug log).

We do use a custom image size that is slightly larger than the default. From the README, I thought that it might mean that we would need to use ResponsiveCustomSizedImage() instead, but that caused all kinds of different problems. No content loading on the pages, pages with password protected images instead, and so on.

After a bit of trial and error, and your post here- http://www.zenphoto.org/support/topic.php?id=1007440#post-1834935, I removed the “getImageTitle()” and that got the page working.

While the page worked, we were getting this in the debug log for every hit on the image page-

Zenphoto v1.4.10[removed]

WARNING: Missing argument 1 for printHDDefaultSizedImage(), called in /Applications/MAMP/htdocs/site/themes/basic_site/image.php on line 80 and defined in /Applications/MAMP/htdocs/site/plugins/zp_picturefill.php on line 640

printHDDefaultSizedImage called from include (image.php [80])

from include (index.php [129])

from index.php [28]

{21213:Wed, 28 Oct 2015 11:29:51 GMT}

NOTICE: Undefined variable: imgobj in /Applications/MAMP/htdocs/site/plugins/zp_picturefill.php on line 642

printHDDefaultSizedImage called from include (image.php [80])

from include (index.php [129])

from index.php [28]

So changing the entry (as you describe here- http://www.zenphoto.org/support/topic.php?id=1007440#post-1834937) to printHDDefaultSizedImage(null, true) stopped the error messages from showing up.

It would be great if this info was added to the README, so other people don’t have a similar issue.

**Comment by acrylian** [comment]
Thanks, I will look into the notices as soon as I can. 

The readme is just meant as the basic info actually and not hte full documentation. The detailed info about the functions itself is documented directly within the `zp_picturefill.php` file itself.  That should always be up to date. 

As you see the intro doc block is already different as the readme as I tend to forget to update it…

**Comment by acrylian** [comment]
I have updated the `$imgobj` parameter so it has a default `null` which some function had missing. So you now can call the HD functions in the right image context without any parameters set as `$hd` is set true by default.
